vast/extension.go

```diff
@@ -3,5 +3,16 @@ package vast
 // Extension type represents an <Extension> element that contains arbitrary XML data provided by the
 // platform to extend the VAST response.
 type Extension struct {
-	Data []byte `xml:",innerxml"`
+	Data          []byte          `xml:",innerxml"`
```

Comment:
Do we need this field? It's causing test issues with marshalling then unmarshalling.

Reply:
Doesn't look like it's used anywhere.